engine/schema/schemeable.test.ts

```diff
@@ -77,7 +77,7 @@ Deno.test("Simple type generation", async () => {
     fail("SimpleType should exists");
   }
 
-  const name = Deno.build.os === "windows" ? "tl@157-178" : "tl@155-174";
+  const name = "tl@155-174";
```

Comment:
Will this pass even in different OSs?

Reply:
I'm fixing now so don't break the build. But my check is broken locally, even with my Deno.build.os returning "windows"